package.json

```diff
@@ -12,7 +12,9 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "lint": "tslint -p .",
-    "deploy": "npm run build && func azure functionapp publish agid-functions-app-test"
+    "deploy": "npm run build && func azure functionapp publish agid-functions-app-test",
```

Comment:
we should remove this task from package.json

Reply:
Right... I know. I decided to leave that task for 1 or 2 weeks to avoid creating problems in the next days if anybody has to use it for any reason (?). 
Also the CircleCI configuration file and directory will need to be removed (now I only disabled the corresponding Webhook and deselected the status checks to pass in master from the GitHub repository settings).

Reply:
nobody is using it moreover the functionapp name is wrong, so it's absolutely better to remove it asap

Reply:
@gunzip Ok. Now I also noticed a task in package.json (`prestart`) that includes the command `func extensions install`... this task/command is not used in the current Azure pipeline to prepare the build for the deployment. Do you know if we have to use it? 
https://github.com/pagopa/io-functions-app/blob/2d425e77b9d52d0a1b4e394d836b6ba7a94b4410/package.json#L7-L9

Reply:
no we don't. func is a local development client